### DISCUSSION TITLE
Instead of spaces, can a root site also be migrated?

### DISCUSSION THREAD

**Discussion by haroldvandekamp** [WikiTransformationProject/wikitraccs-releases#41] [Q&A] [answered]
In the example quick start video, all the spaces are enumerated and a space is selected to be migrated. Is it also possible to migrate the root Confluence site?

**Comment by heinrich-ulbricht** [comment]
@haroldvandekamp I'm not sure I fully understand. Which Confluence version are you referring to? Which content would be missing when you selected all existing spaces for migration?

**Comment by haroldvandekamp** [comment]
@heinrich-ulbricht : Taking https://wiki.hyperledger.org as an example. Take https://wiki.hyperledger.org as an example Confluence site. I see that it is possible to migrate all spaces, like those listed under https://wiki.hyperledger.org/spacedirectory/view.action, e.g. https://wiki.hyperledger.org/display/CASIG/ and https://wiki.hyperledger.org/display/CA/. For me it's unclear whether the root site https://wiki.hyperledger.org can also be migrated. Is the root site itself also a space and in this case is https://wiki.hyperledger.org/display/HYP/Hyperledger the equivalent to https://wiki.hyperledger.org?

**Comment by heinrich-ulbricht** [comment, marked as answer]
@haroldvandekamp Ah I see what you mean. Yes, that root is a space itself. Taking wiki.hyperledger.org as sample again - it uses the **HYP** space:

![screenshot-cursor](https://github.com/WikiTransformationProject/wikitraccs-releases/assets/3469970/c02c14c6-0ec7-42fb-a51d-926bfd32e5da)

I suppose these instructions from Atlassian apply: [Configuring the Site Home Page](https://confluence.atlassian.com/doc/configuring-the-site-home-page-138140.html). It involves selecting the space to use.

**Comment by haroldvandekamp** [comment]
@heinrich-ulbricht : Thanks for the information. That is all clear.
